src/auro-dropdown.js

```diff
@@ -94,7 +106,32 @@ class AuroDropdown extends LitElement {
     this.dropdownWidth = this.getBoundingClientRect().width;
   }
 
+  /**
+   * @private
+   * @returns {void} Updates the trigger `aria-label` or `aria-labelledby`.
+   */
+  processAriaLabel() {
+    this.trigger = this.shadowRoot.querySelector(`#trigger`);
```

Comment:
This is a duplicate of line 100 - can we remove this line and this function call after the trigger is set?

Reply:
We also set the trigger in the `firstUpdated` method, where this method is called from. We should only set it once